platform/linux-generic/include/odp/api/plat/debug_inlines.h

```diff
@@ -54,6 +54,10 @@ extern odp_abort_func_t _odp_abort_fn;
 		 _ODP_LOG_FN(level, "%s:%d:%s(): " fmt, __FILE__, \
 		 __LINE__, __func__, ##__VA_ARGS__)
 
+#define _ODP_LOG_PREFIX(level, prefix, fmt, ...) \
```

Comment:
Having the prefix also in debug entries could make it easier to search relevant stuff from logs.

Reply:
Changed in V3.